specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/longTermRetention.json

```diff
@@ -0,0 +1,723 @@
+        "serverName": {
+          "description": "The server name that the backup database belong to.",
+          "type": "string",
+          "readOnly": true
+        },
+        "serverCreateTime": {
+          "format": "date-time",
+          "description": "The create time of the server.",
+          "type": "string",
+          "readOnly": true
+        },
+        "databaseName": {
+          "description": "The name of the database the backup belong to",
+          "type": "string",
+          "readOnly": true
+        },
+        "databaseDeletionTime": {
+          "format": "date-time",
+          "description": "The delete time of the database",
+          "type": "string",
+          "readOnly": true
+        },
+        "backupTime": {
+          "format": "date-time",
+          "description": "The time the backup was taken",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "Resource": {
+      "description": "ARM resource.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Resource ID.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "Resource name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "Resource type.",
+          "type": "string",
+          "readOnly": true
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "ProxyResource": {
+      "description": "ARM proxy resource.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ]
+    },
+    "LongTermRetentionBackup": {
+      "description": "A long term retention backup.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/LongTermRetentionBackupProperties",
+          "description": "Resource properties.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "LongTermRetentionBackupListResult": {
+      "description": "A list of long term retention bacukps.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Array of results.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LongTermRetentionBackup"
+          },
+          "readOnly": true
+        },
+        "nextLink": {
+          "description": "Link to retrieve next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "LongTermRetentionPolicyProperties": {
+      "description": "Properties of a long term retention policy",
+      "type": "object",
+      "properties": {
+        "weeklyRetention": {
+          "description": "The weekly retention policy for an LTR backup in an ISO 8601 format.",
```

Comment:
add format- date-time for all the 3 properties below.

Reply:
This is ISO **duration** format. e.g. "P1W" representing 1 week duration. Not actually a datetime value.